zebra/zebra_vty.c

```diff
@@ -1020,65 +1013,87 @@ DEFUN (show_ipv6_nht_vrf_all,
 	return CMD_SUCCESS;
 }
 
-DEFUN (ip_nht_default_route,
+DEFPY (ip_nht_default_route,
```

Comment:
Why do we need to change these from DEFUN to DEFPY?  We are not taking advantage of any of the features here.